front/pages/w/[wId]/ds/index.tsx

```diff
@@ -168,6 +321,110 @@ export default function DataSourcesView({
+              use.
+            </p>
+          </div>
+        </div>
+
+        <div className="mt-8 overflow-hidden">
+          <ul role="list" className="">
+            {managedDataSourcesLocal.map((ds) => {
+              return (
+                <li
+                  key={`managed-${ds.connectorProvider}`}
+                  className="px-2 py-4"
+                >
+                  <div className="flex items-center justify-between">
+                    <div className="flex items-center">
+                      {ds.enabled ? (
+                        <Link
+                          href={`/w/${
+                            owner.sId
+                          }/ds/${`managed-${ds.connectorProvider}`}`}
+                          className="block"
+                        >
+                          <p
+                            className={classNames(
+                              "truncate text-base font-bold",
+                              ds.enabled ? "text-violet-600" : "text-slate-400"
+                            )}
+                          >
+                            {ds.name}
+                          </p>
+                        </Link>
+                      ) : (
+                        <p
+                          className={classNames(
+                            "truncate text-base font-bold",
+                            ds.enabled ? "text-violet-600" : "text-slate-400"
+                          )}
+                        >
+                          {ds.name}
+                        </p>
+                      )}
+                    </div>
+                    <div>
+                      {!ds.enabled ? (
+                        <Button
+                          disabled={
+                            !ds.isBuilt ||
+                            isLoadingByProvider[ds.connectorProvider]
+                          }
+                          onClick={
+                            canUseManagedDataSources
+                              ? () => {
+                                  handleEnableManagedDataSource(
+                                    ds.connectorProvider as ConnectorProvider
+                                  );
+                                }
+                              : () => {
+                                  logger.info(
+                                    {
+                                      workspace: owner.sId,
+                                      connector_provider: ds.connectorProvider,
+                                    },
+                                    "request_early_access_managed_data_source"
+                                  );
+                                }
+                          }
+                        >
+                          {!ds.isBuilt
+                            ? "Coming soon"
+                            : !canUseManagedDataSources
+                            ? "Get early access"
```

Comment:
@spolu @gabhubert  when a managed data source provider is already built on our end, but the user's workspace isn't whitelisted for managed DS, I replace the button with "Get early access" -- Is that OK ? Any wording improvement suggestion ?

Reply:
SGTM

Ideally if they click we invite to ping us at team@dust.tt. a window.alert is enough